infrasim/model/core/task.py

```diff
@@ -140,6 +140,8 @@ def run(self):
                     self.__print_task(self.get_task_pid(), self.__task_name, "running")
                     self.__logger.info("[ {:<6} ] {} is running".format(self.get_task_pid(),
                                                                         self.__task_name))
+                    if "node" in self.__task_name:
```

Comment:
it leads to failure (no attribute) if user use "***node***" as node name in yaml.
suggest using an empty (abstract) function in task (base class) and implementing a concrete function in CCompute (subclass), which handles the actual process.